docs/core/v3.0.rst

```diff
@@ -271,17 +267,15 @@ The following figure illustrates the first part of the terminology:
 
     Each node in a hierarchy_ has a path which uniquely identifies
     that node and defines its location within the hierarchy_. The path
-    is formed by joining together the "/" character, followed by the
+    is a string, formed by joining together the "/" character, followed by the
     name_ of each ancestor node separated by the "/" character,
     followed by the name_ of the node itself. For example, the path
     "/foo/bar" identifies a node named "bar", whose parent is named
     "foo", whose parent is the root of the hierarchy. The path "/"
     identifies the root node.
 
-    A path always starts with ``/``.
-
-    QUESTION: do we want to codify that group ``path`` end, can end, or must end
-    in ``/`` ?
+    A path always starts with ``/`` and cannot end with ``/``,
```

Comment:
Currently, implementations support specifying the path without the initial slash, e.g. `foo/bar`.

Reply:
I think this is fine to allow it in an API of an implementation, but when such paths are in the spec or metadata I believe it's good to have a more strict guideline. Should we clarify this or do you think it's fine as-is?